var/spack/repos/builtin/packages/giflib/package.py

```diff
@@ -44,6 +44,10 @@ def install_targets(self):
         targets = ['install'] + self.prefix_and_libversion_args()
         return targets
 
+    @property
+    def libs(self):
+        return find_libraries(['libgif'], root=self.prefix, recursive=True)
```

Comment:
Can we avoid recursion here? There are issues when recursion starts from the prefix and we use externals.

Reply:
Hmm, is this sort of a catch-22? It is not known _a priori_ what the library directory is, `lib` or `lib64`, as that is part of what is being determined by `find_libraries`. What would I set the `root` to if not using recursion?

Reply:
@glennpj Can we do something like:
```python
libs = find_libraries(['libgif'], root=self.prefix.lib) or find_libraries(['libgif'], root=self.prefix.lib64)
```
?

Reply:
We could, and I thought of something like above. It probably covers most cases but it seems like there could be libraries in a directory named differently, which would be covered by recursion. I guess I am thinking more generally, not necessarily the giflib case, and perhaps over thinking. 

Thanks.